### PR TITLE
Debug collect - pcap

### PR DESCRIPTION
**Pull request by philipatl**

### CODE REVIEW

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "debug-collect-pcap",
    "component": "enforcerd",
    "pr-id": "1716",
    "commit-sha": "4f927f5b773225dffd8a2a4677c5556f44b7211a",
    "pipeline": "master"
  },
  {
    "project": "debug-collect-pcap",
    "component": "gaia",
    "pr-id": "489",
    "commit-sha": "0dbc67f7bf58c61d454dfc7091b6a47de0d3dc50",
    "pipeline": "master"
  },
  {
    "project": "debug-collect-pcap",
    "component": "apoctl",
    "pr-id": "247",
    "commit-sha": "feebdf7b1aabcb92035943604bfec75e76c86510",
    "pipeline": "master"
  },
  {
    "project": "debug-collect-pcap",
    "component": "backend",
    "pr-id": "734",
    "commit-sha": "ed3c09b605aa90c8f4d943935274adfe3f2281f9",
    "pipeline": "master"
  }
]
```

File: doc/documentation.md [hunk 1]
```diff
@@ -4230,7 +4230,7 @@ Sends a Enforcer Refresh command.
 
 Contains the ID of the target enforcer.
 
-##### `debug` `emum(Counters | Logs | Packets | PUState)`
+##### `debug` `emum(Counters | Logs | Packets | PUState | Pcap)`
```
Comment (brianonn) [inline doc/documentation.md:4233]: emum -> enum :):)

Comment (brianonn) [reply doc/documentation.md:4233]: wait, wot? WTH is an `emum` because that's used 68 times in that document already.

Comment (brianonn) [reply doc/documentation.md:4233]: I still think it should be `enum` everywhere, judging by the context of it's use  :)

Comment (mheese) [reply doc/documentation.md:4233]: @brianonn these are autogenerated anyway :) ... the only thing we really need to look at in this file are the `.spec` file changes.

Comment (primalmotion) [reply doc/documentation.md:4233]: yep typo in rego doc. will fix later

File: specs/enforcerrefresh.spec [hunk 2]
```diff
@@ -40,3 +41,9 @@ attributes:
     exposed: true
     read_only: true
     autogenerated: true
+
+  - name: debugDuration
```
Comment (primalmotion) [inline specs/enforcerrefresh.spec:45]: must add the `$timeDuration` validation.

```
validations:
    - $timeDuration
```

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "debug-collect-pcap",
    "component": "enforcerd",
    "pr-id": "1716",
    "commit-sha": "b552a02d274d55a2998f91408c71a1bf3a33ad7a",
    "pipeline": "master"
  },
  {
    "project": "debug-collect-pcap",
    "component": "gaia",
    "pr-id": "489",
    "commit-sha": "90a3a18552a380f246b6055e76cb5f09933638aa",
    "pipeline": "master"
  },
  {
    "project": "debug-collect-pcap",
    "component": "apoctl",
    "pr-id": "247",
    "commit-sha": "feebdf7b1aabcb92035943604bfec75e76c86510",
    "pipeline": "master"
  },
  {
    "project": "debug-collect-pcap",
    "component": "backend",
    "pr-id": "734",
    "commit-sha": "952f3bd6d522ce227e1de49477370afa27814ee9",
    "pipeline": "master"
  }
]
```

**Pull request merged**